src/content-script/components/toast/toast.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
I'm keeping this file here because I want to create a new ticket to override `react-toast-notification`'s style.

You up for this? @stnichol